import-service/handlers/importProductsFile.js

```diff
@@ -0,0 +1,35 @@
+import middy from "@middy/core";
+import cors from "@middy/http-cors";
+import { autoProxyResponse } from "middy-autoproxyresponse";
+
+import s3Factory from "../utils/s3Factory";
+import { BUCKET, UPLOADED_FOLDER} from "../constants";
+
+const handler = async event => {
+  const { queryStringParameters: { name } } = event;
+  const filePath = `${UPLOADED_FOLDER}${name}`;
+
+  const s3 = s3Factory.getS3();
+  const params = {
+    Bucket: BUCKET,
+    Key: filePath,
+    Expires: 60,
+    ContentType: 'text/csv'
```

Comment:
would be great to add support for other csv types such as `application/vnd.ms-excel`
from Windows I constantly get `403 Forbidden` because Chrome automatically sets content type as `application/vnd.ms-excel` instead of `text/csv`